SubProject.cmake

```diff
@@ -225,8 +229,13 @@ if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/.gitsubprojects")
         "else()\n"
         "  file(APPEND .gitsubprojects \"git_subproject(${__subproject})\n\")\n"
         "endif()\n")
+      set(__subproject_paths "${__subproject_paths} ${__subproject_dir}")
```

Comment:
replace
set(__subproject_paths "${__subproject_paths} ${__subproject_dir}")
with
list(APPEND __subproject_paths ${__subproject_dir})
and remove separate_arguments(__subproject_paths)?